### PR TITLE
fix: Interface replacement: obtain schema, configure query

### PR DESCRIPTION
**Pull request by KaiMing-L**

### CODE REVIEW

File: client/src/domains-core/graph-analysis/graph-schema/components/config-query/index.tsx [hunk 1]
```diff
@@ -103,44 +104,33 @@ const ConfigQuery: React.FC = () => {
...
     const values = await form.validateFields();
+ 
+
     const { label, property, value, logic, limit = 10, hasClear } = values;
 
     tabContainerField.setComponentProps({
       spinning: true,
     });
 
-    const graphData = await runQuickQueryGraphData(
```
Comment (Seventysevendays) [inline client/src/domains-core/graph-analysis/graph-schema/components/config-query/index.tsx:112]: 直接删掉

File: client/src/pages/graph-schema/index.tsx [hunk 2]
```diff
@@ -82,23 +78,19 @@ const GraphSchema: React.FC = () => {
     //     });
     //   }
     // });
-    // runGetGraphSchema({
```
Comment (Seventysevendays) [inline client/src/pages/graph-schema/index.tsx:85]: 删掉

File: client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  });
+
+  return newProperties;
+};
+
+/* Graph analysis schema data conversion */
+export const graphSchemaTranslator = (schema: Schema[]) => {
+  const nodes: AnalysisSchema[] = [];
+  const edges: AnalysisSchema[] = [];
+  schema?.map(item => {
```
Comment (Seventysevendays) [inline client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts]: foreach

File: client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { AnalysisSchema, Schema, SchemaProperty } from '@/types/services';
+/* properties conversion  */
+const propertiesTranslator = (properties: SchemaProperty[] | undefined) => {
```
Comment (Seventysevendays) [inline client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts]: 拆出去

File: client/src/domains-core/graph-analysis/graph-schema/components/config-query/index.tsx (hunk 1 above)
Comment (KaiMing-L) [reply client/src/domains-core/graph-analysis/graph-schema/components/config-query/index.tsx:112]: fixed

File: client/src/pages/graph-schema/index.tsx (hunk 2 above)
Comment (KaiMing-L) [reply client/src/pages/graph-schema/index.tsx:85]: 还没替换完相关逻辑，暂时保留，替完后会删掉～

File: client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts (hunk 3 above)
Comment (KaiMing-L) [reply client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts]: fixed

File: client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts (hunk 4 above)
Comment (KaiMing-L) [reply client/src/domains-core/graph-analysis/graph-schema/translators/graph-schema-translator/index.ts]: fixed

**Pull request merged**